library/std/src/sys/sgx/abi/tls.rs

```diff
@@ -87,18 +87,21 @@ impl Tls {
     }
 
     pub unsafe fn activate(&self) -> ActiveTls<'_> {
-        set_tls_ptr(self as *const Tls as _);
+        // SAFETY: External function.
```

Comment:
This doesn't appear to explain why it's safe to call? I would at least want a FIXME if we can't document these functions appropriately.